### PR TITLE
Add some more stuff to your list

### PR DESCRIPTION
**Pull request by bridgetbarnes**
It's obviously not exhaustive, but a full list is beyond the scope of any mortal

### CODE REVIEW

File: www-bucket/script.js [hunk 1]
```diff
@@ -12,7 +12,20 @@ const triggers = [
...
+  'proxy cards',
+  'items in smash',
+  'being thumbsd down',
+  '541',
+  'gorilla warfare',
+  'being emailed about the hub',
+  'sausage sizzle',
+  'multipart emojis',
+  'snapchat',
+  ''
```
Comment (72636c) [inline www-bucket/script.js]: ?

File: www-bucket/script.js [hunk 2]
```diff
@@ -12,7 +12,20 @@ const triggers = [
...
-  'tri-state Booleans'
+  'tri-state Booleans',
+  'gates in edh',
+  'burgers',
+  '9 letter puzzles',
+  'proxy cards',
+  'items in smash',
+  'being thumbsd down',
+  '541',
+  'gorilla warfare',
```
Comment (72636c) [inline www-bucket/script.js]: I would recommend some 🦍 use

**Review by 72636c — APPROVED** [review]
🎉

File: www-bucket/script.js (hunk 1 above)
Comment (bridgetbarnes) [reply www-bucket/script.js]: I was trying to think of more to add and forgot I wrote this, removing

File: www-bucket/script.js (hunk 2 above)
Comment (bridgetbarnes) [reply www-bucket/script.js]: This is why you're a senior

File: www-bucket/script.js [hunk 3]
```diff
@@ -12,7 +12,19 @@ const triggers = [
...
   'Okta',
   'rubber domes',
-  'tri-state Booleans'
+  'tri-state Booleans',
+  'gates in edh',
+  'burgers',
+  '9 letter puzzles',
+  'proxy cards',
+  'items in smash',
+  'being thumbsd down',
```
Comment (72636c) [inline www-bucket/script.js]:
```suggestion
  'being 👎ed down',
```

**Review by 72636c — APPROVED** [review]

File: www-bucket/script.js [hunk 4]
```diff
@@ -12,7 +12,19 @@ const triggers = [
...
+  'gates in edh',
+  'burgers',
+  '9 letter puzzles',
+  'proxy cards',
+  'items in smash',
+  'being 👎d down',
+  '541',
+  '🦍 warfare',
+  'being emailed about the hub',
+  'sausage sizzle',
```
Comment (Jack-023) [inline www-bucket/script.js:25]: This is ok. _Sausage sandwich_ is wrong.

File: www-bucket/script.js [hunk 5]
```diff
@@ -12,7 +12,19 @@ const triggers = [
   "Merge branch 'master' into branch",
   'Okta',
   'rubber domes',
-  'tri-state Booleans'
+  'tri-state Booleans',
+  'gates in edh',
+  'burgers',
+  '9 letter puzzles',
+  'proxy cards',
```
Comment (Jack-023) [inline www-bucket/script.js:19]: Proxy cards are totally fine imo.

Comment (bridgetbarnes) [reply www-bucket/script.js:19]: So why'd you give me real cards

Comment (Jack-023) [reply www-bucket/script.js:19]: Proxies are fine but real cards are nicer. When your deck is built around a card and the promo foil version is $1 you might as well get the bling version.

**Pull request merged**